cmd/prom-aggregation-gateway/main.go

```diff
@@ -261,11 +261,12 @@ func (a *aggate) handler(w http.ResponseWriter, r *http.Request) {
 func main() {
 	listen := flag.String("listen", ":80", "Address and port to listen on.")
 	cors := flag.String("cors", "*", "The 'Access-Control-Allow-Origin' value to be returned.")
+	pushPath := flag.String("push-path", "/metrics/", "HTTP path to accept pushed metrics.")
 	flag.Parse()
 
 	a := newAggate()
 	http.HandleFunc("/metrics", a.handler)
```

Comment:
This clashes with the `pushPath` default value. Are you sure this doesn't break it?

Reply:
It's a breaking change, yes.